packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -487,26 +506,26 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     const id = this._id;
     const isItemSelected =
       item.index !== undefined && selectedIndices ? selectedIndices.indexOf(item.index) > -1 : false;
-    const checkboxStyles = () => {
-      return getCheckboxStyles({
-        theme: getTheme(),
-        checked: isItemSelected,
-        disabled: item.disabled
-      });
-    };
+
+    // select the right classname based on the combination of selected/disabled
+    const itemClassName =
+      isItemSelected && item.disabled === true // preciate: both selected and disabled
+        ? this._classNames.dropdownItemSelectedAndDisabled
+        : isItemSelected // preciate: selected only
+          ? this._classNames.dropdownItemSelected
+          : item.disabled === true // predicate: disabled only
+            ? this._classNames.dropdownItemDisabled
+            : this._classNames.dropdownItem;
 
     return !this.props.multiSelect ? (
       <CommandButton
         id={id + '-list' + item.index}
-        ref={Dropdown.Option + item.index}
+        ref={DropdownBase.Option + item.index}
```

Comment:
Shouldn't thins be componentRef? 

`ref` will simply return the first react component this resolves to. If CommandButton is wrapped in an HOC (e.g. `styled` or `createComponent`), we won't get the expected interface.

Or, we investigate and use `fowardRef` in all HOCs.

Reply:
@dzearing So, I didn't actually rationalize this thing at all (the change is a mechanical F2 rename refactor) and I did not touch any JS logic. Your comment puzzled me though because I would have expected the code to have already broken if it was hitting the HOC (makes sense). Turns out that as far as I can tell, we're not using it so I have removed this ref and the corresponding static.

I also went ahead to add more screener tests :)